tests/jballerina-unit-test/src/test/resources/testng.xml

```diff
@@ -252,11 +252,7 @@
                     <exclude name="testAnonStructAsStructField"/>
                 </methods>
             </class>
-            <class name="org.ballerinalang.test.object.ObjectTypeReferenceTest">
-                <methods>
-                    <exclude name="testInvalidTypeReferenceAcrossModules"/>
-                </methods>
-            </class>
+            <class name="org.ballerinalang.test.object.ObjectTypeReferenceTest"/>
```

Comment:
I think these are already included in https://github.com/ballerina-platform/ballerina-lang/pull/26797/files#diff-3835b6074874bfd7bcb2e191247640239cbc9c4546822113a142766502ade096R122

Reply:
Removed